datafusion/core/tests/sqllogictests/postgres/test_files/simple_aggregation.slt

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#   100 100 7.8100000000000000 781 125 -117     --- postgres
+#   100 100 7.81 781 125 -117                   --- datafusion
+
+query IIRIII
+SELECT
+  count(*) AS count_all,
+  count(c3) AS count_c3,
+  round(avg(c3), 0) AS avg,
```

Comment:
I added a `round` here because Postgres produces a text average `7.8100000000000000`, but Datafusion gives `7.81`. 
Treating such collisions explicitly is a drawback of using sqllogictest text-based results comparison.

Reply:
See my comment on https://github.com/apache/arrow-datafusion/issues/4462 about potentially normalizing prior to converting to string

Reply:
Normalization makes sense to me. I just wasn't sure if text representation is expected to be the same.

I'll need to use the "extended" Postgres client then https://github.com/risinglightdb/sqllogictest-rs/blob/main/sqllogictest-bin/src/engines/postgres_extended.rs.

Anyway, introducing normalization is totally doable.

Reply:
I rewrote the Postgres client and added normalization of typed results. The same conversions are applied to Datafusion.